packages/qdrant-loader/src/qdrant_loader/connectors/confluence/connector.py

```
@@ -792,162 +804,256 @@ def _clean_html(self, html: str) -> str:
+                                logger.debug(
+                                    f"Processed {content['type']} '{content['title']}' "
+                                    f"(ID: {content['id']}) from space {self.config.space_key}"
                                 )
+                        except (ValueError, KeyError, AttributeError, TypeError) as e:
+                            logger.error(
+                                "Failed to process Confluence content",
+                                content_type=content.get("type"),
+                                content_title=content.get("title"),
+                                content_id=content.get("id"),
+                                error=str(e),
+                            )
```

Ruff (0.15.17)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)